internal/slack/users_from_conversation.go

```diff
@@ -30,8 +27,7 @@ func (s *client) getChatMembers(conversation Conversation) ([]data.User, error)
 func (s *client) getAllUsers(id string) ([]data.User, error) {
 	users, err := s.getAllUsersFromConversation(id)
 	if err != nil {
-		s.log.Errorf("failed to get all users from chat")
-		return nil, err
+		return nil, errors.Wrap(err, "failed to get all users from chat")
```

Comment:
It seems to me that `getChatMembers` and `getAllUsers` do nothing :)

Reply:
Also nice comment about priority queue in requests with pagination, It's not urgent, but we should keep in mind it and fix later.